<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewAccountsWindow</class>
 <widget class="QMainWindow" name="ViewAccountsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AccountGuard - View Accounts Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/qrc/app-icon.qrc">
    <normaloff>:/newPrefix/imgs/lock-logo.ico</normaloff>:/newPrefix/imgs/lock-logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/imgs/bg.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="viewAccountsWindow_BackBtnFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="viewAccountsWindow_BackBtn">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #527C00;
	border-image: none;
	border: none;
	color: white;
}

QPushButton::hover {
	background-color: #335413;
}</string>
         </property>
         <property name="text">
          <string>  â®œ   BACK  </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="viewAccountsWindow_HeadingFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>119</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="viewAccountsWindow_HeadingLabel">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>View your accounts.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="addAccountWindow_TextFrame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="addAccountWindow_TextLabel1">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Below are the current accounts that you have stored into your local database.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="viewAccountsWindow_TypesList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color: white;
	border-image: none;
}</string>
    </property>
   </widget>
   <widget class="QListWidget" name="viewAccountsWindow_EmailsList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color: white;
	border-image: none;
}</string>
    </property>
   </widget>
   <widget class="QListWidget" name="viewAccountsWindow_PasswordsList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color: white;
	border-image: none;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewAccountsWindow_SectionLabel1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: white;
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="text">
     <string>Account Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="viewAccountsWindow_SectionLabel2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: white;
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="text">
     <string>Account Email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="viewAccountsWindow_SectionLabel3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: white;
	background-color: none;
	border-image: none;
}</string>
    </property>
    <property name="text">
     <string>Account Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="viewAccountsWindow_UnselectBtnFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="viewAccountsWindow_UnselectBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #79B600;
	border-image: none;
	border: none;
	color: white;
}

QPushButton::hover {
	background-color: #527C00;
}</string>
         </property>
         <property name="text">
          <string>Unselect Items</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>viewAccountsWindow_BackBtn</tabstop>
  <tabstop>viewAccountsWindow_TypesList</tabstop>
  <tabstop>viewAccountsWindow_EmailsList</tabstop>
  <tabstop>viewAccountsWindow_PasswordsList</tabstop>
 </tabstops>
 <resources>
  <include location="../assets/qrc/background-gradient.qrc"/>
  <include location="../assets/qrc/app-icon.qrc"/>
 </resources>
 <connections/>
</ui>
